app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from abc import abstractmethod, ABC
+from typing import Any
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> Any:

```

Comment:
`Any` may be possible but we know for a fact which type is returned in each child class so you can be more specific